src/rgw/rgw_auth.cc

```diff
@@ -376,7 +376,7 @@ void rgw::auth::WebIdentityApplier::create_account(const DoutPrefixProvider* dpp
   rgw_apply_default_bucket_quota(user->get_info().bucket_quota, cct->_conf);
   rgw_apply_default_user_quota(user->get_info().user_quota, cct->_conf);
 
-  int ret = user->store_info(dpp, null_yield, true);
+  int ret = user->store_by_id(dpp, null_yield, true);
```

Comment:
odd that id isn't a parameter in such a method?

Reply:
the form of the name s/store_info/store_by_id/; seems just cosmetic--what is the motivation for it, in consideration of the above;  ok, I assume it relies on an id member;  but is there any other kind of load/store that you envision?

Reply:
It was to go with the load_by_id() call.